factcast-store/src/main/resources/db/changelog/factcast/issue2116/notify_schemastore_change.sql

```diff
@@ -0,0 +1,39 @@
+-- noinspection SqlNoDataSourceInspectionForFile
+
+-- looks like pg may starve if we're replacing a function used in a trigger that is busy.
+-- so even if we'd risk to loose a notify, we're now dropping the trigger first
+
+DROP TRIGGER IF EXISTS tr_deferred_schemastore_insert ON schemastore;
```

Comment:
inser makes actually sens?
unify this approach - on emergencies, I want to this also on delete